src/actions/TransactionActions.js

```diff
@@ -1500,3 +1501,55 @@ export const generateEchoAddress = (label) => async (dispatch, getState) => {
 		return null;
 	}
 };
+
+
+export const changeDelegate = (delegateId) => async (dispatch, getState) => {
```

Comment:
input is not clear in interval between sending and broadcast tx.
and need error when account already parent